projects/angular-resizable-element/src/lib/resizable.directive.ts

```diff
@@ -323,7 +323,9 @@ export class ResizableDirective implements OnInit, OnDestroy {
       tap(({ event }) => {
         if (currentResize) {
```

Comment:
I think we can simplify this a bit by moving the `event.cancelable` into this if statement like so:

```
if (currentResize && event.cancelable) {
```

WDYT?

Reply:
Yep, I agree. I've done that but not committed just yet because... 

I did think about removing the try/catch, given that preventDefault() doesn't get called if it can't be cancelled. Keep that, or remove that too...?

Reply:
Oh yeah great shout, I think we can just go ahead and remove the try catch now as well 😄